client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1183,7 +1183,7 @@
   "label.toggle-password-visibility": "Toggle password visibility",
   "label.total": "Total",
   "label.total-before-tax": "Total before tax",
-  "label.total-cost-price": "Total cost",
+  "label.purchase-cost-price": "Purchase cost",
```

Comment:
Should CIV feedback be applied to english translations? I like having things coherent, so have moved both to 'purchase cost'